types/prosemirror-view/index.d.ts

```diff
@@ -546,7 +550,7 @@ export interface EditorProps<S extends Schema = any> {
     [name: string]: (
       node: ProsemirrorNode<S>,
       view: EditorView<S>,
-      getPos: () => number,
+      getPos: () => number | boolean,
```

Comment:
You need some parens here:

```typescript
      getPos: (() => number) | boolean,
```

(`() => number | boolean` is the type of functions that do not require any arguments and return a `number` or a `boolean`.)

Reply:
Good spot 😄